skimage/exposure/exposure.py

```diff
@@ -341,10 +374,15 @@ def rescale_intensity(image, in_range='image', out_range='dtype'):
     array([  0,  63, 127], dtype=int8)
 
     """
-    dtype = image.dtype.type
+    input_dtype = image.dtype.type
```

Comment:
This is only used once now, maybe move it into the if-clause or remove the assignment altogether?